village/src/App.js

```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
-
 import './App.css';
 import SmurfForm from './components/SmurfForm';
 import Smurfs from './components/Smurfs';
+import axios from 'axios';
+
+const theSmurfsURL = 'http://localhost:3333/smurfs';
```

Comment:
good use of const to hold the URL what is the benefit of using a const here over using a var?

Reply:
var is the old syntax and const is the new ES6 and it makes code easily readable ....something along the lines it cant be changed when you assign it a variable